src/main/java/salesforce/ui/pages/opportunity/newopportunity/NewOpportunityLightningPage.java

```diff
@@ -0,0 +1,314 @@
+    @FindBy(css = "h2[class^='inlineTitle slds-p-top--large slds-p-horizontal--medium']")
+    private WebElement newOpportunityTitle;
+
+    @FindBy(xpath = "//div//label//span[text()='Opportunity Name']/../..//input")
+    private WebElement opportunityNameTextBox;
+
+    @FindBy(xpath = "//div//label//span[text()='Account Name']/../..//input")
+    private WebElement accountNameTextBox;
+
+    @FindBy(xpath = "//label//span[text()='Account Name']/../..//div//ul//span[@class='pillText']")
+    private WebElement accountItem;
+
+    @FindBy(xpath = "//span[text()='Amount']/../..//input")
+    private WebElement amountTextBox;
+
+    @FindBy(xpath = "//span[text()='Next Step']/../..//input")
+    private WebElement nextStepTextBox;
+
+    @FindBy(xpath = "//span[text()='Probability (%)']/../..//input")
+    private WebElement probabilityTextBox;
+
+    @FindBy(xpath = "//span[text()='Primary Campaign Source']/../..//input")
+    private WebElement campaignTextBox;
+
+    @FindBy(xpath = "//label//span[text()='Primary Campaign Source']/../..//div//ul//span[@class='pillText']")
+    private WebElement campaignItem;
+
+    @FindBy(xpath = "//span[text()='Order Number']/../..//input")
+    private WebElement orderNumberTextBox;
+
+    @FindBy(xpath = "//span[text()='Main Competitor(s)']/../..//input")
+    private WebElement mainCompetitorTextBox;
+
+    @FindBy(xpath = "//span[text()='Current Generator(s)']/../..//input")
+    private WebElement currentGeneratorTextBox;
+
+    @FindBy(xpath = "//span[text()='Tracking Number']/../..//input")
+    private WebElement trackingNumberTextBox;
+
+    @FindBy(xpath = "//span[text()='Type']/../../div//a[@class='select']")
+    private WebElement typeCombobox;
+
+    @FindBy(xpath = "//span[text()='Lead Source']/../../div//a[@class='select']")
+    private WebElement leadSourceCombobox;
+
+    @FindBy(xpath = "//span[text()='Stage']/../../div//a[@class='select']")
+    private WebElement stageCombobox;
+
+    @FindBy(xpath = "//span[text()='Delivery/Installation Status']/../../div//a[@class='select']")
+    private WebElement deliveryStatusCombobox;
+
+    @FindBy(xpath = "//span[text()='Close Date']/../../div//input[@class=' input']")
+    private WebElement closeDateCalendar;
+
+    @FindBy(css = "textarea[class=' textarea']")
+    private WebElement descriptionTextarea;
+
+    @FindBy(css = "div button[title='Save']")
+    private WebElement saveButton;
+
+    @FindBy(css = "div button[title='Cancel']")
+    private WebElement cancelButton;
+
+    private static final String ACCOUNT_NAME = "//div[@title='%s']";
+    private static final String TYPE_NAME = "//a[@title='%s']";
+    private static final String LEAD_NAME = "//a[@title='%s']";
+    private static final String STAGE_NAME = "//a[@title='%s']";
+    private static final String CAMPAIGN_NAME = "//div[text()='%s']";
+    private static final String DELIVERY_NAME = "//a[@title='%s']";
```

Comment:
I think the locators are repeated. If you can, you could improve this but no problem

Reply:
Yes, but If a put in only one constant. What could be the name?, it is a bit confuse

Reply:
What do think? @Amet015